src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3540,6 +3540,26 @@
     public static final int COMMENT_CONTENT =
             GeneratedJavaTokenTypes.COMMENT_CONTENT;
 
+    /**
+     * Variable Declarator List.
+     *
+     * <pre>
+     * +--VARIABLE_DEC
```

Comment:
We already have `PARAMETERS` as the parent of a `PARAMETER_DEF`. Let's call it `VARIABLES`.